metadata.json

```diff
@@ -20,15 +28,17 @@
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
         "11",
-        "12"
+        "12",
+        "15"
       ]
     },
     {
       "operatingsystem": "OpenSUSE",
       "operatingsystemrelease": [
         "11",
         "12",
-        "13"
+        "13",
+        "15"
```

Comment:
Update supported OSes is not expected with this PR.
It is clearer to create a dedicated PR.

Reply:
Changed.